app/views/layouts/_header.html.erb

```diff
@@ -78,6 +78,15 @@
           <li><%= link_to t("layouts.about"), about_path, :class => "dropdown-item" %></li>
         </ul>
       </li>
+      <li class="nav-item">
+        <% if current_user && current_user.id %>
+          <%= link_to(preferences_path, :class => "nav-link text-secondary ms-1") do %>
```

Comment:
You have `ms-1` for disabled selector icons but no margin for enabled icons.

Reply:
I updated it to be consistent for both situations.